### ISSUE TITLE
Highlighting different activities in different colors

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#316]
When a user is logged in to their home page, all of their bids (submissions to act, teach, volunteer, or vend) are displayed in their rightmost column.

Similarly, scheduled activities (class, panels, rehearsal slots, performance times, volunteer shifts) are displayed in their center column.

It'd be nice to have some way of distinguishing the different categories by color, which would also connect them visually no matter which column they showed up in.  For example, if the classes that someone submitted were displayed in a box with a blue border (or between two blue HRs) in the right column, then classes would appear in the same blue text on their schedule.

We'd have to strike that careful balance between communicating information effectively, and looking like a clown orgy was taking place, but I think it's possible.  Low priority.

**Comment by jonkiparsky** [comment]
Example css for this on demo. It's a bit clown orgy at the moment, feel free to improve it. 

Center needs a little work before we can differentiate types.

**Comment by jonkiparsky** [comment]
@burlexpo  Example css for this on demo. It's a bit clown orgy at the moment, feel free to improve it.
Center needs a little work before we can differentiate types.

**Comment by jonkiparsky** [comment]
@burlexpo  Note that I have not yet committed my changes to base.css, in order to avoid this styling accidentally showing up on the live site. You can edit the base.css on demo directly (in emacs) or use sftp to grab a copy and put your changes.

**Comment by burlexpo** [comment]
It looks like the colors are being applied to just the links, and not the total line.
Compare the live site to the demo.  On the live site, it says "Mr. Scratch (Click to edit)"; on the demo site it says

Mr. Scratch  < highlighted
(Click to edit)  < not highlighted

Because of the template, I don't want to mess with the .css too much on this on my own.

**Comment by jonkiparsky** [comment]
I have  a lot of flexibility on where I set these divs. Can you specify how you'd like it to be? (feel free to give me a call, I'm up for a while)
